processes/bootstrap/bootstrap.conf

```diff
@@ -0,0 +1,10 @@
+processes:
+  - name: "vpp"
+  log-file-path: "/tmp/bootstrap.log"
```

Comment:
This is not valid YAML. Try using some online validator, it throws: `Error: You cannot define a mapping item when in a sequence`.

The lines 3-6 and 8-10 should be indented together with `name`.

Reply:
IDE aligned it when copied